### DISCUSSION TITLE
Can't view aligned reads

### DISCUSSION THREAD

**Discussion by kuffelgr** [GMOD/jbrowse-components#2460] [Q&A] [answered]
Hey all, I'm working with with JBrowse2 1.5.0, the Desktop app. When I load my assembly and then my .bam and .bai files I don't see any reads at locations where I know there are reads. I can see these reads in the Integrative Genome Viewer (similar app from the Broad) using the same exact files. Any ideas? I've tried many things, but nothing seems to work.

**Comment by cmdcolin** [comment, marked as answer]
Just to follow up there was a good discussion in gitter channel about this, mainly centered around configuring the refNameAliases e.g. translation of 1 to chr1 or vice versa

This is currently something users who are loading their own assembly have to do manually but this was actually missing from v1.5.0 of jbrowse desktop so we are working to fix that in the next release
